src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -250,4 +250,12 @@ object TezosDatabaseOperations extends LazyLogging {
   def doBlocksExist(): DBIO[Boolean] =
     Tables.Blocks.exists.result
 
+  /** count number of the elements in the table */
+  def count(table: TableQuery[_]): DBIO[Int] =
```

Comment:
just thinking, `countRows` might be more explicit?

Reply:
sure, why not 😄